src/coreclr/inc/clrconfigvalues.h

```diff
@@ -790,6 +790,10 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableArm64Sha256,            W("EnableArm64Sh
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableArm64Rcpc,              W("EnableArm64Rcpc"),           1, "Allows Arm64 Rcpc+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableArm64Rcpc2,             W("EnableArm64Rcpc2"),          1, "Allows Arm64 Rcpc2+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableArm64Sve,               W("EnableArm64Sve"),            1, "Allows Arm64 SVE hardware intrinsics to be disabled")
+#if defined(TARGET_LINUX) && (defined(DEBUG) || defined(_DEBUG))
+// TODO-SVE: Once coreclr supports vector lengths >16bytes, the default should be -1.
+RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_MaxVectorLength,           W("MaxVectorLength"),          16, "Specifies maximum size of the vector length in bytes while using SVE on Arm64", CLRConfig::LookupOptions::ParseIntegerAsBase10)
```

Comment:
Why can't we just use `MaxVectorTBitWidth` for this?

Reply:
Right, I understand what this PR is trying to fix.

What I'm saying is that we already have an option to control the maximum size of `Vector<T>` and that is `DOTNET_MaxVectorTBitWidth`, so why aren't we just using that to influence whether or not `prctl` is called instead?

Arm64 should really just have a bit of code that looks vaguely like this: https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/codeman.cpp#L1274-L1287, where it:
1. Queries the current SVE vector length
2. Compares that against `MaxVectorTBitWidth`
3. If the user has explicitly set a maximum width and its less than the current bit width, we then have the option of either disabling SVE (undesirable) or using the OS functionality (such as `prctl`) to match the user targeted size
4. In the default experience where the user hasn't explicitly set a maximum width (it is `0`), then long term we should do nothing but short term we should use the OS functionality to fix the size to 128-bits, until the support for larger widths can be brought online

Reply:
Makes sense, I'll move this code to use `MaxVectorTBitWidth`. Thanks for this suggestion 👍

Reply:
@SwapnilGaikwad - are you planning to push this patch as well as remove it from `#ifdef DEBUG` in `codeman.cpp`?

Reply:
Yup, removed the changes from `#ifdef DEBUG` as they now relay on `MaxVectorTBitWidth`.